overlord/devicestate/devicemgr.go

```diff
@@ -144,7 +147,10 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 			return nil, err
 		}
 		if modeEnv != nil {
+			logger.Debugf("modeenv for model %s found", modeEnv.Model)
```

Comment:
nitpick: just noticed that on one comment below you spelt this as "modeEnv".

Reply:
Right, in general it looks like modeenv is preferred, so I'll change things in that direction.

Reply:
But you didn't change it yet, did you?

Reply:
I did for this concrete variable! But not for the full PR, I've done that now :)